UIKitMovie/UIKitMovie/View/FilmViewController.swift

```diff
@@ -0,0 +1,314 @@
+        static let descriptionLabelBottomIndent = 0.0
+        static let descriptionLabelLeftIndent = 20.0
+        static let descriptionLabelRightIndent = 20.0
+        static let descriptionLabelTopAnchorConstant = 10.0
+        static let descriptionLabelLeftAnchorConstant = 5.0
+        static let descriptionLabelRightAnchorConstant = -5.0
+        static let descriptionLabelBottomAnchorConstant = -40.0
+        static let backButtonCornerRadius = 10.0
+        static let backButtonBorderWidth = 1.0
+        static let backButtonTitle = "Назад"
+        static let backButtonHeightAnchorConstant = 30.0
+        static let backButtonWidthAnchorConstant = 50.0
+    }
+
+    // MARK: Private Visual Component
+
+    private let filmImageView = UIImageView()
+    private let filmOriginLabel = CustomLabel()
+    private let filmFeatureLabel = CustomLabel()
+    private let filmDescriptionLabel = CustomLabel()
+    private let backButton = UIButton(type: .roundedRect)
+
+    // MARK: - Public Properties
+
+    var film: Movie?
+    var presenter: FilmViewPresenterProtocol!
+
+    // MARK: - Life Cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+    }
+
+    // MARK: - Private Methods
+
+    private func setupUI() {
+        presenter.setFilm()
+        createVisualPresentation()
+        createFilmImageView()
+        createFilmOriginLabel()
+        createFilmFeatureLabel()
+        createFilmDescriptionLabel()
+        createBackButton()
+        createFilmImageViewConstraint()
+        createFilmOriginLabelConstraint()
+        createFilmFeatureLabelConstraint()
+        createFilmDescriptionLabelConstraint()
+        createBackButtonConstraint()
+    }
+
+    private func createVisualPresentation() {
+        view.backgroundColor = Constants.blackCustomColor
+        title = film?.title
+        navigationController?.navigationBar
+            .titleTextAttributes = [NSAttributedString.Key.foregroundColor: Constants.yellowCustomColor as Any]
+        navigationController?.navigationBar.tintColor = Constants.yellowCustomColor
+        navigationItem.rightBarButtonItem = UIBarButtonItem(
+            image: UIImage(systemName: Constants.rightBarButtonItemImageName),
+            style: .plain,
+            target: self,
+            action: #selector(rightBarButtonAction)
+        )
+    }
+
+    private func createFilmImageView() {
+        view.addSubview(filmImageView)
+        filmImageView.backgroundColor = Constants.blackCustomColor
+        filmImageView.layer.cornerRadius = Constants.filmImageViewCornerRadius
+        filmImageView.clipsToBounds = true
+        filmImageView.contentMode = .scaleToFill
+        filmImageView.layer.borderWidth = Constants.filmImageViewBorderWidth
+        filmImageView.layer.borderColor = Constants.yellowCustomColor?.cgColor
+        guard let image = film?.backdropPath else { return }
+        filmImageView.loadImage(image, networkService: NetworkService())
```

Comment:
```suggestion
        filmImageView.loadImage(image, networkService: )
```
достаем через презентер